types/pg-pool/index.d.ts

```diff
@@ -1,7 +1,28 @@
-// Type definitions for pg-pool
+// Type definitions for pg-pool 2.0
 // Project: https://github.com/brianc/node-pg-pool
-// Definitions by: Leo Liang <https://github.com/aleung>
+// Definitions by: Leo Liang <https://github.com/aleung>, Nikita Tokarchuk <https://github.com/mainnika>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-/// <reference types="pg" />
-export {Pool, PoolConfig } from "pg";
+import * as pg from 'pg';
+
+declare class Pool<T extends pg.Client> extends pg.Pool {
+    readonly Client: Pool.ClientLikeCtr<T>;
+
+    constructor(config?: Pool.Config<T>, client?: Pool.ClientLikeCtr<T>);
+
+    connect(): Promise<T & pg.PoolClient>;
+    connect(callback: (err: Error, client: T & pg.PoolClient, done: (release?: any) => void) => void): void;
```

Comment:
Needs `query` ? https://github.com/brianc/node-pg-pool#your-new-favorite-helper-method

Reply:
Implemented in the `pg` types, https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/pg/index.d.ts#L158